apps/concierge_site/config/config.exs

```diff
@@ -62,6 +62,14 @@ config :concierge_site, mailchimp_api_key: {:system, "MAILCHIMP_API_KEY", ""}
 config :concierge_site, mailchimp_list_id: {:system, "MAILCHIMP_LIST_ID", "abc123"}
 config :concierge_site, mailchimp_api_client: ConciergeSite.Mailchimp.FakeClient
 
+# Informizely
+config :concierge_site,
+  informizely_site_id: {:system, "INFORMIZELY_SITE_ID", "69f607f3-3826-4aa6-9f86-996a07670833"}
+
+config :concierge_site,
+  informizely_account_deleted_survey_id:
+    {:system, "INFORMIZELY_ACCOUNT_DELETED_SURVEY_ID", "rhluhvlu"}
+
```

Comment:
question (non-blocking): would it be more appropriate for these to live in [config/runtime.exs](https://github.com/mbta/alerts_concierge/blob/main/config/runtime.exs)?

Are the default values checked in to support local testing?

Reply:
Yeah, that's better, moved them. I also removed the default values. I guess that was the reason, but it doesn't really seem necessary.